### PR TITLE
fix: solving user_menu discrepancies and making enhancements

### LINKED ISSUE #1747: [UI]: `Account Tab` under docs is ambiguos.
**Describe the bug**

`Account Tab` which is shown in dropdown of `profile` in docs is ambiguos as it redirects to `dashboard` rather than `client information`

**To Reproduce**
1. Go to 'http://127.0.0.1:5080/docs/'
2. Click on 'Profile Icon'
3. Scroll down to 'Account'
4. Click on 'Account'
5. See error

**Expected behavior**

There should be a dropdown available for `Dashboard` and `Account`.

**Screenshots**

Click on account **First Time**:
image

Click on account **Second Time**
image

**Platform (please complete the following information):**
 - OS: Linux
 - Browser: Chrome

### PR DESCRIPTION
**Pull request by vishakhanihore**
Fixes #1747

### CODE REVIEW

**Comment by djmitche** [comment]
Not a review, but a comment: I like this change as I find myself often clicking my user icon in a place where the menu comes up with "sign out" under my mouse, and accidentally clicking again signs me out.  This will nicely move that item out of the way of my sloppy clicking :)

**Comment by vishakhanihore** [comment]
@helfi92 @djmitche A bug which I couldn't resolve, After this changes the `UserMenu` on docs section has also been updated, But the `Profile` still redirects to `/` first, whereas I have described it to redirect to `/profile`. I can't debug it.
Any reason why is this behaviour present?

**Comment by vishakhanihore** [comment]
> @helfi92 @djmitche A bug which I couldn't resolve, After this changes the `UserMenu` on docs section has also been updated, But the `Profile` still redirects to `/` first, whereas I have described it to redirect to `/profile`. I can't debug it.
> Any reason why is this behaviour present?

Figured it out, No problem !

**Comment by vishakhanihore** [comment]
@helfi92 @djmitche This is ready for review 😄

**Comment by vishakhanihore** [comment]
@helfi92 @djmitche This is ready for review 😄

File: ui/src/components/UserMenu/index.jsx [hunk 1]
```diff
@@ -106,14 +107,18 @@ export default class UserMenu extends Component {
           anchorEl={anchorEl}
           open={Boolean(anchorEl)}
           onClose={this.handleMenuClose}>
+          <MenuItem title="Your Dashboard" component={Link} to="/">
+            <HomeAnalyticsIcon className={classes.leftIcon} />
+            Dashboard
+          </MenuItem>
```
Comment (helfi92) [inline ui/src/components/UserMenu/index.jsx]: No need for that. We rely on the convention of clicking the logo to go back to the home page. Sometimes, we only want users to view the documentation site (e.g., https://docs.taskcluster.net).

File: ui/src/components/UserMenu/index.jsx [hunk 2]
```diff
@@ -106,14 +107,18 @@ export default class UserMenu extends Component {
...
           <MenuItem title="Your Profile" component={Link} to="/profile">
             <AccountIcon className={classes.leftIcon} />
-            Account
+            Profile
           </MenuItem>
           <MenuItem
             title={`Sign Out of ${window.env.APPLICATION_NAME}`}
             onClick={this.handleSignOutClick}>
-            <HandPeaceIcon className={classes.leftIcon} />
+            <LogoutVariantIcon className={classes.leftIcon} />
```
Comment (helfi92) [inline ui/src/components/UserMenu/index.jsx]: I like this icon :)

File: ui/src/App/docsRoutes.jsx [hunk 3]
```diff
@@ -14,6 +17,11 @@ export default [
     component: Documentation,
     path: join(DOCS_PATH_PREFIX, ':path*'),
   },
+  // Defining Profile Route for correct redirection
+  {
+    component: Profile,
+    path: '/profile',
+  },
```
Comment (helfi92) [inline ui/src/App/docsRoutes.jsx]: Ah, that makes a lot of sense!

File: ui/src/App/docsRoutes.jsx [hunk 4]
```diff
@@ -14,6 +17,11 @@ export default [
     component: Documentation,
     path: join(DOCS_PATH_PREFIX, ':path*'),
   },
+  // Defining Profile Route for correct redirection
```
Comment (helfi92) [inline ui/src/App/docsRoutes.jsx]: No need for that comment :)

**Comment by vishakhanihore** [comment]
Got it, will be reverting an extra menu list.

**Comment by vishakhanihore** [comment]
@helfi92 Please Review

**Comment by vishakhanihore** [comment]
@helfi92 @djmitche Please merge.

**Comment by helfi92** [comment]
Waiting on https://github.com/taskcluster/taskcluster/pull/1729 before merging.

**Pull request merged**